src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -56,7 +56,7 @@ public AddCommand parse(String args) throws ParseException {
 
         // Allow setting optional parameter roomNumber in add command
         // Do not allow setting optional parameter emergencyContact in add command
-        Person person = new Person(name, phone, email, roomNumber, address, null, tagList);
+        Person person = new Person(name, phone, email, roomNumber, address, null, null, tagList);
```

Comment:
Would it be better to add a comment explaining that gradYear is also not an optional parameter in add command?